ovs/lib/scheduledtask.py

```diff
@@ -263,11 +269,10 @@ def gather_scrub_work():
                 local_scrub_location = scrub_info[1]
```

Comment:
Local scrub is not executed in another task, it's executed in the same task by the running worker so it can't remain in PENDING.